scripts/latest-posts.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Performs the initialization of the community feeds for the home page.
+ */
+async function initialize_community_feeds() {
+    await get_latest_news();
+    await get_latest_community();
+}
+
+/**
+ * Gets the latest news posts from the monogame discourse and injects
+ * it into the #latest-community element on the page
+ */
+async function get_latest_news() {
+    let response = await fetch('https://communityapi.aristurtle.net/latest-news');
+    let json = await response.json();
+
+    if (json.topic_list.topics != undefined) {
+        let latestNewsContainer = document.getElementById("latest-news");
+
+        for (let i = 0; i < json.topic_list.topics.length; i++) {
+            let news = json.topic_list.topics[i];
+            let latestNewsWrapper = document.createElement('div');
+            latestNewsWrapper.className = 'col-12 mb-3';
+            latestNewsWrapper.appendChild(create_post_block(news));
+            latestNewsContainer.appendChild(latestNewsWrapper);
+        }
+    }
+}
+
+/**
+ * Gets the latest community posts from the monogame discourse and injects
+ * it into the #latest-community element on the page
+ */
+async function get_latest_community() {
+    let response = await fetch('https://communityapi.aristurtle.net/latest-community');
```

Comment:
@AristurtleDev was it your intent to leave this linked to your own site?

Maybe now that this is published to the official MG site the direct community API link will work?